### ISSUE TITLE
Unable to connect to desktop

### ISSUE THREAD

**Issue by samcro1967** [Ylianst/MeshCentral#2790]
I have the latest version of MeshCentral installed (0.8.65) on a Windows 10 desktop.  I have one agent installed on a Windows 10 machine that is also running Hyper-V.  It has a vNIC for the virtual switch.  I am unable to connect to the desktop.  The agent is reporting and I I can connect to it via the terminal.  I do not see any errors in the event log.  The hostname for the desktop is the NIC for the hyper-v switch.  Not sure if that also means that is the NIC is is trying to use to connect to the desktop.  I am able to connect to another Windows 10 machine that only has one NIC.

Not sure what to look at or try next.

**Comment by VINISHVV** [comment]
Are you able to access your  Meshcentral  webpage from the agent PC using any browser ?. 
Also Please let me know whether your server is running in LAN  / WAN / Hybrid mode.

**Comment by samcro1967** [comment]
It was running in LAN mode and I was able to connect to the web page just fine from both clients I tested with agents.  I tried 3 different docker containers as well and could not get either of the 2 agents to connect to any of the containers.  I have dozens of other apps running in Docker so not sure what the issue was.  Going to explore other alternatives.  This doesn't seem to work in my environment.  Thanks for the follow up.
